Cargo.toml

```diff
@@ -15,7 +15,7 @@ build = "src/gl/build.rs"
 name = "gl"
 path = "src/gl/lib.rs"
 
-[build-dependencies.gl_generator]
+[dependencies.gl_generator]
```

Comment:
The tests wouldn't build otherwise. Should I add it under `build-dependencies` and `dev-dependencies`?